DirectEmployers/MyJobs
# [PD 3125] add contact display cbo column

## What's this PR do?
PR Adds the CBO details which is JUST the CBO Name to the right hand facet side when adding an additional contact inside of the manage cbo page

## Screenshots (if appropriate)
![screen shot 2017-04-10 at 1 15 47 pm](https://cloud.githubusercontent.com/assets/22029077/24917043/e9d387d2-1ea9-11e7-8c57-38a8a69396ec.png)


## Where should the reviewer start?
Start inside of the CBO Application

## How should this be manually tested?
Pull down the repository. Go into a CBO and on the manage CBO page you should see Add Additional Contact. When clicked, you should see a facet on the right hand side that has the CBO
name of the current CBO that you are adding an additional contact for

## Any background context you want to provide?
